markets/perps-market/contracts/storage/PerpsAccount.sol

```diff
@@ -397,10 +358,7 @@ library PerpsAccount {
         // update market data
         perpsMarket.updateMarketSizes(oldPositionSize, position.size);
```

Comment:
We should emit `MarketUpdated` at some point after this happens. It is currently only emitted when positions are settled. This means market size, skew, and price can be updated during a liquidation without logging those changes.

Reply:
YES! good catch.